### ISSUE TITLE
Document science data products and product metadata

### ISSUE THREAD

**Issue by lewismc** [capstone-coal/capstone-coal.github.io#34]
Once we have addressed https://github.com/capstone-coal/pycoal/issues/118 we should really think about documenting the science data products in a formal way such that consumers are crystal clear on what to expect and how to process.

**Comment by ghost** [comment]
Agreed. Let me know if you need any feedback on the data formats that were used. Small details like the unsigned 16-bit integer data type that were chosen for the environmental correlation output. I have complained a lot about libraries that make you dig into the source code to figure out what is going on, so I don't want to add to the problem!

It would also be helpful to document more clearly what happens during each processing stage. As I review the capstone materials for the AGU presentations I am coming across the sketches and outlines we shared. More polished and up-to-date versions of these could be used to explain the algorithm in a visual way. I suppose that informal documentation is the opposite of what you created this issue for, so another could be created for that.

There is also Issue https://github.com/capstone-coal/capstone-coal.github.io/issues/28 on documenting our San Juan case study which is not currently detailed on the website.

**Comment by lewismc** [comment]
Once we have all of this material together (hopefully before Christmas time) I'll begin working on our next publication which will significantly expand upon the BiDS'17 paper and which we will submit for blind peer reviewed journal publication.
